src/main/java/edu/tamu/sage/service/SimpleProcessorService.java

```diff
@@ -49,6 +49,12 @@ public class SimpleProcessorService implements ProcessorService {
     @Value("${app.solr.batch-size:250}")
     private Integer batchSize;
 
+    @Value("${app.googleTag}")
+    private String googleAnalytics;
+
+    public String getGoogleAnalytics() {
+        return googleAnalytics;
+    }
```

Comment:
Please don't put the getter here.

relocate it to after all of the properties are defined, such as after line 53 below.